pkg/controller/directvolumemigration/stunnel.go

```diff
@@ -51,12 +63,27 @@ data:
     client = yes
     syslog = no
     [rsync]
-    accept = {{ .StunnelPort}}
+    accept = {{ .StunnelPort }}
     CAFile = /etc/stunnel/certs/ca.crt
     cert = /etc/stunnel/certs/tls.crt
+{{ if not (eq .ProxyHost "") }}
+    protocol = connect
+    connect = {{ .ProxyHost }}
+    protocolHost = {{ .RsyncRoute }}:443
```

Comment:
missing this if missing an`{{ end }}`  or am I misreading?

Reply:
youre misreading. the {{ end }} is after the else block